zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -950,9 +976,16 @@ benchmarks! {
             Pallet::<T>::dispute(
                 RawOrigin::Signed(origin).into(),
                 market_id,
-                OutcomeReport::Scalar(i.into())
+                OutcomeReport::Scalar((i % 2).into())
```

Comment:
For authorized, we will only allow one dispute. But doesn't hurt if you have proven the benchmarking logic with the current main.

Reply:
Yeah, much of this will go out the window when `Disputes` is removed, but right now, it makes sense to me to update this.